app/services/ConfigAgentTrait.scala

```diff
@@ -54,4 +55,17 @@ class ConfigAgent(val config: ApplicationConfiguration, val frontsApi: FrontsApi
 
   def getConfig(id: String): Option[CollectionConfig] = configAgent.get().flatMap(_.collections.get(id).map(CollectionConfig.fromCollectionJson))
 
+  def getFrontsPermissionsPriorityByCollectionId(id: String): Set[PermissionsPriority] = configAgent.get().map(config => {
+    val fronts = config.fronts
+    fronts.foldLeft(Set[PermissionsPriority]())((acc, front) => {
```

Comment:
I think you can destructure in the lambda arguments to avoid using `_2`:

```scala
fronts.foldLeft(Set.empty[PermissionsPriority]) { case(acc, (_, front)) =>
    if(front.collections.contains(id)) {
       ...
    }
}
```